prow/sidecar/options.go

```diff
@@ -75,6 +75,16 @@ type Options struct {
 	// longer than the `entrypoint` grace period for the test process and the time
 	// taken by `sidecar` to upload all relevant artifacts.
 	IgnoreInterrupts bool `json:"ignore_interrupts,omitempty"`
+
+	// Secrets are paths to directories containing secret data. The contents of these
+	// secret data files will be censored from the logs and artifacts uploaded to the
+	// cloud.
+	Secrets []string `json:"secrets,omitempty"`
```

Comment:
Maybe `SecretDirectories`? This requires reading the godoc to understand what is supposed to get in there